sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -257,7 +257,14 @@ const eph = EventProcessorHost.createFromConnectionString(
 // In V2, you get a single event passed to your callback. If you had asynchronous code running in your callback,
 // it is not awaited before the callback is called for the next event.
 const onMessage = (context, event) => {
-  /** your code here **/
+  /** Your code to process the event here **/
+
+  try {
+    await context.checkpointFromEventData(event);
```

Comment:
This might convey that we recommend checkpointing every event. Can we add some pretty words to clarify that?

Reply:
Sure, though honestly I'm not sure what the recommended approach is given there's no backpressure in old EPH!